### PR TITLE
add description on creating a frontend project with cookieplone an…

### PR DESCRIPTION
**Pull request by animus888**
…d add some howtos (add-ons, theme, logo)

## Description
documentation part on creating a project with cookiecutter-volto and add some howtos (add-ons, theme, logo).

### CODE REVIEW

**Review by davisagli — COMMENTED** [review]
Thanks for working on documentation, but I'm worried it could be confusing to add docs about a tool that's in experimental status. I think the general plan is to make cookiecutter-plone-starter use cookiecutter-volto once we're confident that that works well, and then it won't be necessary to use cookiecutter-volto directly unless you're in the specific situation of wanting a new frontend to use with an existing backend.

**Comment by animus888** [comment]
@davisagli Thank you for the information.
May I ask why it is planned in future that users should work with cookiecutter-volto through cookiecutter-plonestarter ?
Or are there any special reasons why users shouldn't use cookiecutter-volto directly?
I just thought it would be an effort to get cookiecutter-volto working through cookiecutter-plonestarter.
Wouldn't it be easier to simply replace cookiecutter-plonestarter with cookiecutter-volto in future?

**Comment by stevepiercy** [comment]
@davisagli the purpose of this PR, as discussed at the Alpine Sprint, is to publish a how-to guide for `cookiecutter-volto`. We settled on Plone 6 Documentation, but with a clear "experimental" indicator. This provides the following advantages.

1.  It gives `cookiecutter-volto` exposure that it otherwise would not get, especially to newcomers.
2.  It reuses includes from other parts of the documentation for pre-requisites.
3.  It reuses the documentation infrastructure.
4.  We don't want to replace the current Install instructions, but documenting this alternative can help us better inform what direction the cookiecutters ultimately take. Documentation will be a single place to run comparisons.

There was strong sentiment expressed at the Alpine Sprint that `cookiecutter-volto` would be the way of the future because of its simpler implementation, but you bring up a valid point which sows uncertainty that should be resolved before we commit to this direction. Clearly there needs to be further discussion about the future of cookiecutters amongst interested parties, and a road map should be agreed upon. I hope that happens, but not in this PR. Can you suggest who should be involved in that discussion, and perhaps take it to the Community Forum? Off the top of my head, I think that would include the following people and possibly others:

- @jensens 
- @davisagli 
- @ericof 
- @sneridagh 

Of course, and as always, we can revise and change tack to the changing winds. I want all of us to sail in the same direction on this ship.

**Comment by jensens** [comment]
IMO it is favorable to have the cookiecutters documented separately. 

1. cookiecutter-plone-starter is supposed to be - as the name tells - a starter. But it is blown up a lot for this purpose btw. So, in fact its no more the small starter but overwhelms people and makes Plone (my opinion) look more complex than it is.
1. Then cookiecutter-volto is not only about bootstrapping a project, but also for addons and themes. So, I do not see why  we would want to not teach people how to use it. 
1. I am totally fine using it within the starter. As a tool.
1. The cookiecutter-volto on its own offers a full development environment for Volto. No starter needed, simple backend start in Docker. It reduces the complexity and lowers the entry barrier a lot. 
1. Last but not least I am a fan of small tools doing one thing.

File: docs/install/create-project-cookiecutter-volto.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  html_meta:
+    "description": "Create a project with cookiecutter-volto (experimental)"
+    "property=og:description": "Create a project with cookiecutter-volto (experimental)"
+    "property=og:title": "Create a project with cookiecutter-volto (experimental)"
+    "keywords": "Plone, Plone 6, create, project, install, cookiecutter-volto, pnpm"
+---
+
+(create-a-project-with-cookiecutter-volto-experimental-label)=
+
+# Create a project with cookiecutter-volto (experimental)
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
# Create a project with `cookiecutter-volto` (experimental)
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+This chapter describes the usage of the currently experimental {term}`cookiecutter` project [plone/cookiecutter-volto](https://github.com/plone/cookiecutter-volto/).
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
This chapter describes the usage of the currently experimental {term}`cookiecutter` project [`cookiecutter-volto`](https://github.com/plone/cookiecutter-volto/).
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+The development of this add-on is done in isolation using a new approach using {term}`pnpm` workspaces and latest {term}`mrs-developer` and other Volto core improvements.
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: It won't be new or latest for long. ;)

```suggestion
The development of this add-on is done in isolation using {term}`pnpm` workspaces, {term}`mrs-developer`, and other Volto core improvements.
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+In order to get started you will need {term}`pipx`, {term}`Make`, {term}`Node.js` and {term}`Docker` see {ref}`create-a-project-with-cookiecutter-volto-experimental-installation-label` section below.
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: I would move this under Prerequisites below.

```suggestion
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+After that you can run following command to create your project:
+```shell
+pipx run cookiecutter gh:plone/cookiecutter-volto
+```
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: I would remove the lines 18-21. We don't want to maintain commands in two places. I also think it is important to put prerequisites before instructions.

File: docs/install/create-project-cookiecutter-volto.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-label)=
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
(create-a-project-with-cookiecutter-volto-experimental-prerequisites-label)=
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+
+## Installation
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: Windows does not support Make out of the box, and there is no way that I want to try to support all the ways of installing it other than through WSL.

I added a couple of other not-so obvious pre-requisites, an operating system that we support (no Android, iOS, etc.), Python, and Git.

````suggestion
## Prerequisites

The following items are required to create a project with `cookiecutter-volto`.

```{include} ../volto/contributing/install-operating-system.md
```

-   Python {SUPPORTED_PYTHON_VERSIONS}
-   {term}`pipx`
-   {term}`nvm`
-   {term}`Node.js` LTS 20.x
-   {term}`GNU make`
-   {term}`Docker`
-   {term}`Git`
````

File: docs/install/create-project-cookiecutter-volto.md [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-pipx-label)=
+
+### pipx
+
+Install {term}`pipx`.
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: We need to have a link to the glossary for each term only once.

```suggestion
Install pipx.
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+    ```shell
+    source ~/.bash_profile
+    ```
+
+4.  Verify that the `nvm` version is that which you just installed or updated:
+
+    ```shell
+    nvm --version
+    ```
+#### Finally install Node.js
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion

(create-a-project-with-cookiecutter-volto-experimental-installation-node.js-label)=

### Node.js
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 10]
```diff
@@ -0,0 +1,183 @@
...
+### GNU make
+
+```{include} ../volto/contributing/install-make.md
+```
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-node.js-label)=
+
+### Node.js
+
+First you'll need nvm, {term}`Node Version Manager`.
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 11]
```diff
@@ -0,0 +1,183 @@
...
+(create-a-project-with-cookiecutter-volto-experimental-installation-gnu-make-label)=
+
+### GNU make
+
+```{include} ../volto/contributing/install-make.md
+```
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-node.js-label)=
+
+### Node.js
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 12]
```diff
@@ -0,0 +1,183 @@
...
+```
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-gnu-make-label)=
+
+### GNU make
+
+```{include} ../volto/contributing/install-make.md
+```
+
+(create-a-project-with-cookiecutter-volto-experimental-installation-node.js-label)=
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: Let's move this to the correct order.

```suggestion
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 13]
```diff
@@ -0,0 +1,183 @@
...
+
+### Node.js
+
+First you'll need nvm, {term}`Node Version Manager`.
+
+#### nvm
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
### nvm
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 14]
```diff
@@ -0,0 +1,183 @@
...
+
+    ```shell
+    source ~/.bash_profile
+    ```
+
+4.  Verify that the `nvm` version is that which you just installed or updated:
+
+    ```shell
+    nvm --version
+    ```
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: Let's convert this entire section to an include file for easier maintenance, and not do copy paste. The include should go into a separate PR in https://github.com/plone/volto, and updated in both this PR and the volto PR in the following places:

- https://6.docs.plone.org/install/create-project.html#nvm
- https://6.docs.plone.org/volto/contributing/developing-core.html#nvm

I should have done that when I did the other includes, but this one may have been under revision.

File: docs/install/create-project-cookiecutter-volto.md [hunk 15]
```diff
@@ -0,0 +1,183 @@
...
+#### Finally install Node.js
+
+```{include} ../volto/contributing/install-nodejs.md
+```
+
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: We need to install Make, unless the cookiecutter does that for the user.

````suggestion


(create-a-project-with-cookiecutter-volto-experimental-installation-make-label)=

### Make

```{include} ../volto/contributing/install-make.md
```
````

File: docs/install/create-project-cookiecutter-volto.md [hunk 16]
```diff
@@ -0,0 +1,183 @@
...
+{term}`Docker` is needed later to start the plone backend inside a container.
+
+```{include} ../volto/contributing/install-docker.md
+```
+
+
+(project-add-ons-label)=
+
+# Project Add-Ons
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
## Project add-ons
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 17]
```diff
@@ -0,0 +1,183 @@
...
+
+### Docker
+
+{term}`Docker` is needed later to start the plone backend inside a container.
+
+```{include} ../volto/contributing/install-docker.md
+```
+
+
+(project-add-ons-label)=
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
````suggestion


(create-a-project-with-cookiecutter-volto-experimental-create-the-project-label)=

## Create the project

After satisfying the prerequisites, create the project.

```shell
pipx run cookiecutter gh:plone/cookiecutter-volto
```


## Build the frontend and backend

To work on your project, you need to build both the frontend and backend.

```shell
make install
```


(project-add-ons-label)=
````

File: docs/install/create-project-cookiecutter-volto.md [hunk 18]
```diff
@@ -0,0 +1,183 @@
...
+
+# Project Add-Ons
+
+After having initially built frontend and backend using the command `make install`,
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]: Where do we tell users to do this? It should be a separate subsection.

File: docs/install/create-project-cookiecutter-volto.md [hunk 19]
```diff
@@ -0,0 +1,183 @@
...
+you will have the configuration file {file}`volto.config.js` (and other stuff) inside your top add-on folder:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
Now that you have a project and built both the frontend and backend, you will have the configuration file {file}`volto.config.js` (and other stuff) inside your top add-on folder:

```

File: docs/install/create-project-cookiecutter-volto.md (hunk 18 above)
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 20]
```diff
@@ -0,0 +1,183 @@
...
+const addons = ['youraddon'];
+const theme = '';
+
+module.exports = {
+  addons,
+  theme,
+};
+```
+
+In {file}`volto.config.js` you can add add-ons like this:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
In {file}`volto.config.js`, you can add add-ons like this:

```

File: docs/install/create-project-cookiecutter-volto.md [hunk 21]
```diff
@@ -0,0 +1,183 @@
...
+```js
+const addons = ['youraddon','@kitconcept/volto-light-theme'];
+```
+
+You also have to add it to the `dependencies` section of {file}`packages/youraddon/package.json`:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
You also have to add it to the `dependencies` section of {file}`packages/<YOUR_ADD_ON>/package.json`:

```

File: docs/install/create-project-cookiecutter-volto.md [hunk 22]
```diff
@@ -0,0 +1,183 @@
...
+```json
+  "dependencies": {
+    "@plone/components": "1.7.0",
+    "@kitconcept/volto-light-theme" : "^3.2"
+  },
+```
+
+If your desired add-on is a theme addon, you should also change following line in {file}`volto.config.js`:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
If your desired add-on is a theme add-on, you should also change the following line in {file}`volto.config.js`:

```

File: docs/install/create-project-cookiecutter-volto.md [hunk 23]
```diff
@@ -0,0 +1,183 @@
...
+```js
+const theme = '@kitconcept/volto-light-theme';
+```
+
+After all that, you have to execute `make install` again in order to install the new add-on.
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
After all that, you need to run `make install` again to install the new add-on.
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 24]
```diff
@@ -0,0 +1,183 @@
...
+
+(start-frontend-and-backend)=
+
+# Start frontend and backend
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
## Start frontend and backend
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 25]
```diff
@@ -0,0 +1,183 @@
...
+Now you are ready to run your project.
+You can start frontend and backend in separate terminal sessions in the same project directory:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
You must start the frontend and backend in separate terminal sessions from the same working directory, at the root of your project.
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 26]
```diff
@@ -0,0 +1,183 @@
...
+
+```shell
+pnpm start
+```
+
+(change-the-logo-label)=
+
+## Change the Logo
+
+You can change the logo locally or by overriding Logo component of an installed theme add-on e.g. `kitconcept/volto-light-theme`.
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
You can change the logo locally, or by overriding the `Logo` component of an installed theme's add-on, for example, `kitconcept/volto-light-theme`.
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 27]
```diff
@@ -0,0 +1,183 @@
...
+
+`````{tab-set}
+````{tab-item} local
+In order to override the Logo component, you will have to add following file:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
To override the `Logo` component, you will have to add a file {file}`packages/youraddon/src/customizations/components/Logo/Logo.jsx`.
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 28]
```diff
@@ -0,0 +1,183 @@
...
+{file}`packages/youraddon/src/customizations/components/Logo/Logo.jsx`
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 29]
```diff
@@ -0,0 +1,183 @@
...
+
+Then you only have to change the line where the logo gets imported to:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
Then change the line where the logo gets imported:

```

File: docs/install/create-project-cookiecutter-volto.md [hunk 30]
```diff
@@ -0,0 +1,183 @@
...
+```js
+import LogoImage from '../../../../../images/Logo.svg';
+```
+
+Assuming you got your Logo ready in an local images folder like {file}`packages/youraddon/src/images/Logo.svg`
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
This assumes your logo is in a local `images` folder, such as {file}`packages/<YOUR_ADD_ON>/src/images/Logo.svg`.
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 31]
```diff
@@ -0,0 +1,183 @@
...
+````
+
+````{tab-item} volto-light-theme
+Target Add-On Component: `src/components/Logo/Logo.jsx` of `kitconcept/volto-light-theme`
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
Target add-on component: `src/components/Logo/Logo.jsx` of `kitconcept/volto-light-theme`
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 32]
```diff
@@ -0,0 +1,183 @@
...
+
+Of course, this only works if you have already installed the `kitconcept/volto-light-theme` add-on.
+
+In order to override the Logo component of the volto-light-theme package, you will have to add following file:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
To override the `Logo` component of the `volto-light-theme` package, add the file {file}`packages/youraddon/src/customizations/@kitconcept/volto-light-theme/components/Logo/Logo.jsx`.
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 33]
```diff
@@ -0,0 +1,183 @@
...
+{file}`packages/youraddon/src/customizations/@kitconcept/volto-light-theme/components/Logo/Logo.jsx`
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 34]
```diff
@@ -0,0 +1,183 @@
...
+
+Then you only have to change the line where the logo gets imported to:
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
Then change the line where the logo gets imported:
```

File: docs/install/create-project-cookiecutter-volto.md [hunk 35]
```diff
@@ -0,0 +1,183 @@
...
+```js
+import LogoImage from '../../../../../images/Logo.svg';
+```
+
+Assuming you got your Logo ready in an local images folder like {file}`packages/youraddon/src/images/Logo.svg`
```
Comment (stevepiercy) [inline docs/install/create-project-cookiecutter-volto.md]:
```suggestion
This assumes your logo is in a local `images` folder, such as {file}`packages/<YOUR_ADD_ON>/src/images/Logo.svg`.
```

File: docs/install/create-project-cookiecutter-volto.md (hunk 7 above)
Comment (stevepiercy) [reply docs/install/create-project-cookiecutter-volto.md]: I also think we need to include pnpm. See https://6.docs.plone.org/volto/contributing/developing-core.html#pnpm

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I made suggestions to re-order and restructure a few things. We should do at least one more pass, to make sure it flows properly.

**Comment by davisagli** [comment]
@stevepiercy @jensens I think we basically agree on most of how you are thinking about cookiecutter-volto, and I withdraw my objection to adding this documentation. However, I have one request which I'll explain below.

As a general strategy I think it makes sense to have high-quality (tested, documented) scaffolds focused on particular pieces of our stack (i.e. Volto, the Plone backend, a deployment solution). There can even be multiple scaffolds for alternative approaches (i.e. one for a deployment using Docker Swarm, another for a deployment using Kubernetes...)

Then cookiecutter-plone-starter should become a wrapper that calls the other scaffolds as tools, for someone who is new and doesn't know what pieces they need. (This could also be a new cookiecutter if that's easier to accomplish, but I've been thinking about it as an evolution of cookiecutter-plone-starter.)

cookiecutter-plone-starter already uses this wrapper approach for generating a volto project using the yeoman-based generator. cookiecutter-volto is a clear improvement over that (because yeoman is not maintained well these days, and because cookiecutter-volto uses pnpm which makes it easier to develop a Volto addon along with other packages). So I've been assuming that we at least want to update cookiecutter-plone-starter to use cookiecutter-volto.

Anyway, my remaining request about this documentation is that I think it could use a clearer introduction which mentions when it would be appropriate to use cookiecutter-volto instead of the existing install docs based on cookiecutter-plone-starter. cookiecutter-volto is great if you are developing something that is frontend only (and just want to run a non-customized Plone backend via docker) but it doesn't give you everything you need if you also want to customize the backend and deploy the solution. Right?

**Comment by animus888** [comment]
@davisagli @stevepiercy @jensens I added the information that cookiecutter-volto currently only supports frontend development to the introduction see [here](https://github.com/plone/documentation/pull/1649/commits/aa09962ff9195b683103bf5dd8b4b42e6a54a3f7).

**Comment by stevepiercy** [comment]
We are sprinting on the cookiecutters at the Axolote Sprint in Mexico City this week, and there may be more changes. I'll push commits to reflect any changes that arise.

File: docs/install/create-project-cookiecutter-volto.md (hunk 14 above)
Comment (animus888) [reply docs/install/create-project-cookiecutter-volto.md]: @stevepiercy added the pull request for plone/volto [here](https://github.com/plone/volto/pull/5968).

Comment (animus888) [reply docs/install/create-project-cookiecutter-volto.md]: @stevepiercy I am now using the nvm installation include file in create-project.html#nvm
create-project-cookiecutter-volto.html#nvm...
I just need to figure out how to update the volto submodule of this fork... and I will add further examples to create-project-cookiecutter-volto.html...

Comment (animus888) [reply docs/install/create-project-cookiecutter-volto.md]: @stevepiercy  the nvm installation include file is now used in create-project.html#nvm and
create-project-cookiecutter-volto.html#nvm...

Comment (stevepiercy) [reply docs/install/create-project-cookiecutter-volto.md]: @animus888 sorry for the delay. We're awaiting revisions to the new cookiecutter that wraps other cookiecutters before pushing out new documentation of any cookiecutter. See https://github.com/plone/cookieplone. @sneridagh, @ericof, and @davisagli started work on that at the Axolote Sprint, and made significant progress, but I'm not sure where it stands.

File: docs/install/create-project-cookiecutter-volto.md (hunk 18 above)
Comment (animus888) [reply docs/install/create-project-cookiecutter-volto.md]: There is a separate subsection "Build the frontend and backend" now. Additionally, information on differences between frontend and backend is added to this subsection.

**Comment by animus888** [comment]
> We are sprinting on the cookiecutters at the Axolote Sprint in Mexico City this week, and there may be more changes. I'll push commits to reflect any changes that arise.

@davisagli @stevepiercy @jensens 
`pipx run cookieplone volto_addon` throws an [error](https://github.com/plone/cookieplone/issues/12).  
And I expected it to install cookiecutter-volto.

If it really should run cookiecutter-volto for the volto_addon option.
Then I would update my documentation part --> "Create a volto_addon project with cookieplone (experimental)"

**Review by ericof — CHANGES_REQUESTED** [review]
cookiecutter-volto is now deprecated.
To get the same features you should use `pipx run cookieplone fromtend_addon`

**Comment by stevepiercy** [comment]
@animus888 if you can make it to the Beethoven Sprint remotely in Discord, there is a daily stand-up at 9:30 AM CET, and a 24/7 chat channel at https://discord.com/channels/786421998426521600/971310680114360340. There has been more progress on the cookiecutters, and things might change again. It would be good to chat with the core developers on this item, especially because they want feedback on its usage.

**Comment by animus888** [comment]
> @animus888 if you can make it to the Beethoven Sprint remotely in Discord, there is a daily stand-up at 9:30 AM CET, and a 24/7 chat channel at https://discord.com/channels/786421998426521600/971310680114360340. There has been more progress on the cookiecutters, and things might change again. It would be good to chat with the core developers on this item, especially because they want feedback on its usage.

@stevepiercy 
I've seen it too late :crying_cat_face: 
Today I tried re-creating my frontend project using cookieplone with the frontend add-on template and it seems to work so far :+1:

**Review by animus888 — COMMENTED** [review]
@stevepiercy this link [blocks the checks](https://github.com/plone/documentation/actions/runs/9758754040/job/26933980675?pr=1649) for my pull request.. :crying_cat_face:  what else can I do?

**Comment by stevepiercy** [comment]
> @stevepiercy this link [blocks the checks](https://github.com/plone/documentation/actions/runs/9758754040/job/26933980675?pr=1649) for my pull request.. 😿 what else can I do?

It was temporary. It's OK to ignore.

**Comment by animus888** [comment]
> > @stevepiercy this link [blocks the checks](https://github.com/plone/documentation/actions/runs/9758754040/job/26933980675?pr=1649) for my pull request.. 😿 what else can I do?
> 
> It was temporary. It's OK to ignore.

Ok, I will try if the original link works now.

**Comment by stevepiercy** [comment]
@animus888 @davisagli picked up your work in this PR, expanded it in https://github.com/plone/documentation/pull/1714, and I did a thorough walk through and review. I'm closing this PR in favor of that one. Would you please review the other PR and walk through it? I'd appreciate your feedback. Thank you!

**Pull request closed without merging**